main.cpp

```diff
@@ -545,14 +431,126 @@ MOVE THEM to the space below this block comment and put them in numerical order
     simply CUT and PASTE them in the space provided below:
 */
 
+/*
+Thing 1) MIDI Keyboard
+5 properties:
+    1) number of keys (int)
+    2) number of knobs (int)
+    3) number of sliders (int)
+    4) number of buttons (int)
+    5) number of pedals (int)
+3 things it can do:
+    1) transmit digital signal
+    2) control virtual instruments
+    3) trigger loops
 
+Thing 2) Coffee Machine
+5 properties:
+    1) water tank capacity (int)
+    2) bean hopper capacity (int)
+    3) espresso shot volume (int)
```

Comment:
these would probably be expressed in milliliters or liters. If you chose liters, then the underlying type should be a float or double.  as in: water tank capacity: 1.2L, espresso shot volume: 0.25L, etc..
- [ ] revise